pysiaf/tests/makeup_polynomial.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
```

Comment:
Please assemble all the individual test files into one single file `test_polynomial.py`.